vaadin-grid-flow-parent/vaadin-grid-flow/src/main/resources/META-INF/resources/frontend/gridConnector.js

```diff
@@ -926,6 +929,16 @@ import { isFocusable } from '@vaadin/grid/src/vaadin-grid-active-item-mixin.js';
             && validSelectionModes.indexOf(mode) >= 0) {
           selectionMode = mode;
           selectedKeys = {};
+          if (mode === validSelectionModes[0]) { // SINGLE
+            grid.$.table.setAttribute('aria-multiselectable', false);
```

Comment:
Does changing the "aria-multiselectable" attribute somehow affect a11y? I tried different screen readers but didn't notice much difference? Please double-check and if it turns out the attribute doesn't affect anything, we could consider removing it from the Web Component.

Reply:
My blind colleague is still on vacation so I can't test it with him. What difference in announcement did you observe? It should be really small, like single vs multiple select possibility (nothing more tho) but it's possible that not all Screenreader can work with role=grid and aria-multiselectable it's a fairly rare pattern. If you want a good read about table and aria roles I can suggest https://sarahmhigley.com/writing/grids-part2/ to cry yourself into sleep ;)

Reply:
> What difference in announcement did you observe?

I didn't notice any difference. Since @sosa-vaadin is not actively working on this right now, we could merge it as it is and postpone investigating the need for "aria-multiselectable" for later.